internal/librarian/flags.go

```diff
@@ -26,7 +26,7 @@ func addFlagAPI(fs *flag.FlagSet, cfg *config.Config) {
 }
 
 func addFlagAPISource(fs *flag.FlagSet, cfg *config.Config) {
-	fs.StringVar(&cfg.APISource, "api-source", "", "location of googleapis repository. If undefined, googleapis will be cloned to the output")
+	fs.StringVar(&cfg.APISource, "api-source", "https://github.com/googleapis/googleapis", "location of googleapis repository. If undefined, googleapis will be cloned to the output")
```

Comment:
Having a URL default here I think will break other code as it assumes APISource is a path. Example: https://github.com/googleapis/librarian/blob/918507c568622dd4792da9ae175fee6239f33763/internal/librarian/state.go#L82